orc8r/cloud/deploy/terraform/orc8r-helm-aws/k8s.tf

```diff
@@ -18,13 +18,17 @@ resource "kubernetes_namespace" "orc8r" {
 }
 
 resource "kubernetes_namespace" "monitoring" {
+  count = var.orc8r_is_staging_deployment == true ? 0 : 1
+
   metadata {
     name = var.monitoring_kubernetes_namespace
   }
 }
 
 # external dns maps route53 to ingress resources
 resource "helm_release" "external_dns" {
+  count = var.orc8r_is_staging_deployment == true ? 0 : 1
```

Comment:
To clarify, we need to add external DNS entries for the "Green" cluster. I will address this as part of the PR that I will push shortly.